### PR TITLE
rename master -> main branch for github

### PR DESCRIPTION
**Pull request by pesho-ivanov**

### CODE REVIEW

File: guide/src/format/configuration/renderers.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ additional-js = ["custom.js"]
 no-section-label = false
 git-repository-url = "https://github.com/rust-lang/mdBook"
 git-repository-icon = "fa-github"
-edit-url-template = "https://github.com/rust-lang/mdBook/edit/master/guide/{path}"
+edit-url-template = "https://github.com/rust-lang/mdBook/edit/main/guide/{path}"
```
Comment (ehuss) [inline guide/src/format/configuration/renderers.md:108]: mdbook's URL uses master, so I probably wouldn't change this line.

File: guide/src/format/configuration/renderers.md [hunk 2]
```diff
@@ -150,7 +150,7 @@ The following configuration options are available:
 - **edit-url-template:** Edit url template, when provided shows a
   "Suggest an edit" button (which looks like <i class="fa fa-edit"></i>) for directly jumping to editing the currently
   viewed page. For e.g. GitHub projects set this to
-  `https://github.com/<owner>/<repo>/edit/master/{path}` or for
+  `https://github.com/<owner>/<repo>/edit/main/{path}` or for
   Bitbucket projects set it to
   `https://bitbucket.org/<owner>/<repo>/src/master/{path}?mode=edit`
```
Comment (ehuss) [inline guide/src/format/configuration/renderers.md:155]: Does Bitbucket default to master? It seems strange to be inconsistent here.

Comment (pesho-ivanov) [reply guide/src/format/configuration/renderers.md:155]: I just created an empty repo on bitbucket.org and it defaulted to "master" (similar to what I saw in other bitbucket repos).

So, I believe the patch is correct.

**Comment by pesho-ivanov** [comment]
It may be helpful to add a note to check one's own repo branch..

**Pull request closed without merging**